Networking/Networking/Model/SystemPlugin.swift

```diff
@@ -39,6 +39,10 @@ public struct SystemPlugin: Decodable, GeneratedFakeable, GeneratedCopiable {
     ///
     public let networkActivated: Bool
 
+    /// Is the plugin active, i.e. false | true
+    ///
+    public let active: Bool
```

Comment:
nit: just wondering, "active" and "activated" are interchangeable in the WP plugins context right? I'm just more used to "activated" as in wp-admin. After seeing `networkActivated` above, I was expecting this to be `activated` but it's very minor 🙂 

TIL'ed "network active plugins" 😄

Reply:
IKR?! I went back and forth on active vs activated. I ultimately picked active because that is the term used in the JSON in the system status response (inactive_plugins, active_plugins)

https://github.com/woocommerce/woocommerce-ios/blob/15f88d616d56c58935d25953931518ee55050f1d/Networking/Networking/Model/SystemStatus.swift#L32-L33